docs/tutorials/quickstart.md

```diff
@@ -0,0 +1,302 @@
+      [Remote - SSH extension](https://open-vsx.org/extension/jeanp413/open-remote-ssh).
+
+   1. Select **Remote-SSH: Connect to host** from the command palette.
+
+   1. Enter the SSH host:
+
+   1. Enter your password:
+
+   </details>
+
+## Work on some code
+
+After VS Code loads the remote environment, you can select **Open Folder** to
+explore directories in the Docker container or work on something new.
+
+To clone an existing repository:
+
+1. Select **Clone Repository** and enter the repository URL.
+
+   For example, to clone the Coder repo, enter
+   `https://github.com/coder/coder.git`.
+
+   Learn more about how to find the repository URL in the
+   [GitHub documentation](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+1. Choose the folder to which VS Code should clone the repo. It will be in its
+   own directory within this folder.
+
+   Note that you cannot create a new parent directory in this step.
+
+1. After VS Code completes the clone, select **Open** to open the directory.
+
+1. You are now using VS Code in your Coder environment!
+
+## What's next?
+
+Now that you have your own workspace, use the same template to set one up for a
+teammate.
+
+Go to **Templates** and select **Create Workspace** and continue from Step 7 in
+[Configure Coder with a new workspace](#configure-coder-with-a-new-workspace).
+
+After that, you can try to:
+
+- [Customize templates](../templates/customizing)
+- [Enable Prometheus metrics](../admin/prometheus#enable-prometheus-metrics)
+- [Deploy to Google Cloud Platform (GCP)](../platforms/gcp)
+
+## Troubleshooting
+
+### Cannot connect to the Docker daemon
+
+> Error: Error pinging Docker server: Cannot connect to the Docker daemon at
+> unix:///var/run/docker.sock. Is the docker daemon running?
+
+1. Install Docker for your system:
+
+   ```shell
+   curl -sSL https://get.docker.com | sh
+   ```
+
+1. Set up the Docker daemon in rootless mode for your user to run Docker as a
+   non-privileged user:
+
+   ```shell
+   dockerd-rootless-setuptool.sh install
+   ```
+
+   Depending on your system's dependencies, you might need to run other commands
+   before you retry this step. Read the output of this command for further
+   instructions.
+
+1. Assign your user to the Docker group:
+
+   ```shell
+   sudo usermod -aG docker $USER
+   ```
+
+1. Confirm that the user has been added:
+
+   ```console
+   $ groups
+   docker sudo users
+   ```
+
+   - Ubuntu users might not see the group membership update. In that case, run
+     the following command or reboot the machine:
+
+     ```shell
+     newgrp docker
+     ```
+
+### Can't start Coder server: Address already in use
+
+```shell
+Encountered an error running "coder server", see "coder server --help" for more information
+error: configure http(s): listen tcp 127.0.0.1:3000: bind: address already in use
+```
+
+1. Stop the process:
+
+   ```shell
+   sudo systemctl stop coder
+   ```
+
```

Comment:
This is only valid if coder was started as a system service i.e. `sudo systemctl start coder`

Reply:
I think I encountered this when I left an ssh session and came back, but I'll have to doublecheck - I think I tried `coder stop` and nothing happened

Reply:
`coder stop` is to stop a coder workspace while `sudo systemctl stop coder` is to stop the coder service on the host where the coder is running an s system service.